ADAL/src/cache/ADTokenCacheItem.m

```diff
@@ -90,6 +94,14 @@ - (id)copyWithZone:(NSZone*) zone
 
 - (ADTokenCacheKey*)extractKey:(ADAuthenticationError* __autoreleasing *)error
 {
+    if (_storageAuthority)
+    {
+        return [ADTokenCacheKey keyWithAuthority:_storageAuthority
+                                        resource:_resource
+                                        clientId:_clientId
+                                           error:error];
+    }
+    
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3AS1131)